app/conversations/__init__.py

```diff
@@ -32,3 +32,10 @@ def post(self, user=None):
         
         result = SocketService.create_conversation(conversation_id, user['email'], participants)
         return utils.make_response(data = { 'conversationId': conversation_id, 'participants': result })
+
+    def remove(self, user=None):
```

Comment:
Finally, this should be decorated with `authenticated_request` and take in the conversation ID. That way, the request would look something like `PUT /conversation/<conversation_id>` and only be accessible to users who have a valid JWT